Learn-JSON-Fetch-async-await-building-fcc-author-page/index.js

```diff
@@ -0,0 +1,45 @@
+const authorContainer = document.getElementById("author-container");
+const loadMoreBtn = document.getElementById("load-more-btn");
+let startingIndex = 0;
+let endingIndex = 8;
+let authorDataArr;
+
+// replace this API with one hosted by FCC
+fetch("https://fcc-author-api.herokuapp.com/authors")
```

Comment:
Would you like to open a PR to add the API code to https://github.com/freeCodeCamp/demo-projects?

Reply:
I opened a PR here
https://github.com/freeCodeCamp/demo-projects/pull/368

Reply:
Thanks @jdwilkin4, just looked at your PR in the Demo Projects repo and everything looks good at first glance.

Before we review and merge that, though, would it be simpler to host this JSON file in the CDN repo instead? My understanding is that we won't update it much if at all once it's hosted somewhere.

Reply:
We shouldn't need to update it at all.
As long as the campers can still fetch the data from there, then hosting it on the CDN is fine with me.
Should I go ahead and close my demo projects PR?
Which one is the CDN repo?

Reply:
Hi @jdwilkin4, sorry, just seeing this now.

Awesome. thanks for confirming that the data will be static.

Yes, I think the PR in the demo projects repo can be closed.

Here's the link to the CDN repo: https://github.com/freeCodeCamp/cdn

I think we can keep things simple and just create a new directory in `/build/curriculum`, name it something like `news-author-page`, and put the JSON file there.

Reply:
@scissorsneedfoodtoo 

I created the PR here
https://github.com/freeCodeCamp/cdn/pull/181

Reply:
Thanks for adding that JSON file to the CDN repo, @jdwilkin4, and for your review and merge, @naomi-lgbt.

I just swapped the Heroku URL out for the [one served by our CDN](https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json) and everything LGTM.

Reply:
```suggestion
fetch("https://cdn.freecodecamp.org/curriculum/news-author-page/authors.json")
```

Looks like this still needs to be updated?

Reply:
@naomi-lgbt 

Ahh yes. 
Let me update that real quick

Reply:
updated 👍